File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { KokkaiKaigirokuApiSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await KokkaiKaigirokuApiSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- KokkaiKaigirokuApi SDK exists test

local sdk = require("kokkai-kaigiroku-api_sdk")

describe("KokkaiKaigirokuApiSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
